run-nitro-enclaves-acm-tests

```diff
@@ -167,7 +181,7 @@ function test_token_rsa_key {
 
 	out=$($P11NE_CLI describe-device)
 	ttl_init=$($P11NE_CLI describe-device \
-		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token")' \
+		| jq '.Ok.DeviceDescription.tokens[] | select(.label == "test-token-rsa")' \
```

Comment:
Could you explain why was this change necessary?

Reply:
In the very first version of this test script, there was a single `test_token` function. Previous feedback suggested to split the test between two functions, one dealing with tokens initialized from rsa keys, while the second one using tokens initialized from secp3841 keys.

In the initial version, when there was a single function, the token label was called `test-token`, while the second version introduced `TOKEN_LABEL_RSA` and `TOKEN_LABEL_SECP384R1_EC`. These two variables were used in all places but these two jq select operations (which were two errors discovered during testing the pipeline when setting up the CI instance).